main.go

```diff
@@ -94,8 +103,16 @@ func main() {
 		dief("looking for test data: %v\n", err)
 	}
 	for _, name := range remove {
-		if err := os.RemoveAll(name); err != nil {
-			dief("removing %v: %v\n", name, err)
+		err := os.RemoveAll(name);
+		if err != nil {
+			failedToRemove[name] = err
+		}
+	}
+	
+	if len(failedToRemove) > 0 {
+		fmt.Fprintf(os.Stderr, "\n# Failed to remove below item, please remove manually\n")
+		for name, err := range failedToRemove {
```

Comment:
nice, below code did fix the issue when removing file under '.git' file.
i just need to add a bit more code to clean up folder. practicing my golang now.

thanks for the information.

```go
cmd := exec.Command("cmd.exe", "/C", "del", "/Q", "/F", "/S", path)
```